bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RackawareEnsemblePlacementPolicy.java

```diff
@@ -56,18 +56,20 @@ protected RackawareEnsemblePlacementPolicy initialize(DNSToSwitchMapping dnsReso
                                                           int maxWeightMultiple,
                                                           int minNumRacksPerWriteQuorum,
                                                           boolean enforceMinNumRacksPerWriteQuorum,
+                                                          boolean ignoreLocalNodeInPlacementPolicy,
```

Comment:
In the near future we should refactor this method and introduce a single bean or anyhow reduce the number of parameters.
Not blocker for this change